clang/test/CodeGen/attr-target-version.c

```diff
@@ -85,7 +85,21 @@ int hoo(void) {
 }
 
 
+// This should generate one target version but no resolver.
+__attribute__((target_version("default"))) int unused_with_forward_default_decl(void);
+__attribute__((target_version("mops"))) int unused_with_forward_default_decl(void) { return 0; }
 
+// FIXME: If the default declaration follows the non-default definition,
```

Comment:
This bug (if we consider it as such) seems unrelated to this patch, but since I found it it deserves a test at least to monitor what we are currently generating for it.

Reply:
Ouch. Yeah, I think we should consider that a bug.